### ISSUE TITLE
Invalid URI in ECSO8.owl

### ISSUE THREAD

**Issue by SArndt-TIB** [DataONEorg/sem-prov-ontologies#135]
Hi everyone,

rdflib 7.0.0 is reporting an invalid URI in ECSO8.owl: in http://purl.dataone.org/odo/ECSO_00003119 "PLFA concentration" with a space is used to signify a resource.

```
    <!-- http://purl.dataone.org/odo/ECSO_00003119 -->

    <owl:Class rdf:about="http://purl.dataone.org/odo/ECSO_00003119">
        <rdfs:subClassOf rdf:resource="http://purl.dataone.org/odo/ECSO_00000520"/>
        <dc:creator rdf:resource="http://orcid.org/0000-0003-1264-1166"/>
        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2019-10-24T07:58:08Z</dc:date>
        <oboInOwl:hasExactSynonym rdf:resource="PLFA concentration"/>
        <rdfs:label xml:lang="en">phospholipid fatty acid concentration</rdfs:label>
        <skos:altLabel rdf:resource="PLFA concentration"/>
    </owl:Class>
```

rdflib generates a warning for this:

```
WARNING:rdflib.term:http://purl.dataone.org/odo/PLFA concentration does not look like a valid URI, trying to serialize this will break.
```

Protege fixes this to http://purl.dataone.org/odo/PLFA%20concentration but other systems might not. Using camel case could be an alternative. So, thanks for considering and all the best to you!

Susanne

**Comment by mbjones** [comment]
Thanks for the report @SArndt-TIB -- we'll take a look and try to get that patched up.

**Comment by mbjones** [comment]
THe problem was an incorrect `rdf:resource` attribute inside `skos:altLabel`, where the value should have been a literal. This has been fixed in version 0.10.0, which I will upload to bioportal shortly.

`<skos:altLabel>PLFA concentration</skos:altLabel>`

**Comment by SArndt-TIB** [comment]
Hi @mbjones thanks a lot for adressing this so quickly. There is also an rdf:resource in this statement. 
 Shouldn't that also be fixed? Thanks!
```
<oboInOwl:hasExactSynonym rdf:resource="PLFA concentration"/>
```
